### PR TITLE
added hovering example with msf

### PR DESCRIPTION
**Pull request by burrimi**

### CODE REVIEW

File: resource/msf_simulator.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+pose_sensor/pose_noise_scale: 0.0
+pose_sensor/pose_noise_p_wv: 0.0
+pose_sensor/pose_noise_q_wv: 0.0
+pose_sensor/pose_noise_q_ic: 0.0
+pose_sensor/pose_noise_p_ic: 0.0
+pose_sensor/pose_delay: 0.0
+pose_sensor/pose_noise_meas_p: 0.005
+pose_sensor/pose_noise_meas_q: 0.005
+
+pose_sensor/init/q_ic/w: 0.9819
```
Comment (ffurrer) [inline resource/msf_simulator.yaml:22]: I'm still in favour of adding a short comment on what the different params are

File: resource/msf_simulator.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+pose_sensor/init/q_ic/x: 0.1060
+pose_sensor/init/q_ic/y: -0.0343
+pose_sensor/init/q_ic/z: 0.1534
+pose_sensor/init/p_ic/x: 0.03
+pose_sensor/init/p_ic/y: -0.07
+pose_sensor/init/p_ic/z: 0.1
+
+pose_sensor/pose_fixed_scale: true
```
Comment (ffurrer) [inline resource/msf_simulator.yaml:30]: especially here and below

File: package.xml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  <author>Janosch Nikolic</author>
+  <author>Markus Achtelik</author>
+
+  <license>ASL 2.0</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <!-- Dependencies needed to compile this package. -->
+  <build_depend>gazebo_plugins</build_depend>
+  <build_depend>rotors_gazebo_plugins</build_depend>
```
Comment (ffurrer) [inline package.xml]: needed?

**Comment by ffurrer** [comment]
some minor comments, otherwise this lgtm, just check which dependencies we really need, I would rather think about setting rotors_simulator as a run dependency or something...

**Comment by burrimi** [comment]
created issue https://github.com/ethz-asl/rotors_simulator_demos/issues/2

**Pull request merged**